airsonic-main/src/main/resources/org/airsonic/player/i18n/ResourceBundle_en.properties

```diff
@@ -373,6 +373,7 @@ generalsettings.welcometitle=Welcome title
 generalsettings.welcomesubtitle=Welcome subtitle
 generalsettings.welcomemessage=Welcome message
 generalsettings.loginmessage=Login message
+generalsettings.sessiontimeout=Session timeout in minutes. Changes will take effect on a new login
```

Comment:
if it's a string, session timeout will be a duration string, if no postfix then in seconds.
also not sure when changes take effect with the spring boot var, maybe at application restart?